FAQ.md

```diff
@@ -191,3 +191,12 @@ Possible mitigations are:
   kernel so as to use `vmalloc` instead of `kmalloc` for them.
 - Reduce memory pressure on the host.
 
+### Error appears when starting Firecracker.
```

Comment:
nit: maybe rephrase so as to document more errors that can appear when starting Firecracker.

```suggestion
### Firecracker fails to start.
```